src/reflect/value_test.go

```diff
@@ -1,6 +1,7 @@
 package reflect_test
```

Comment:
Are there any tests in `all_test.go` you can uncomment?

Reply:
I did not know to look there. I'll check!

Reply:
I don't think any can be uncommented without significant additions. The most obvious candidate is the conversion tests, but this would require implementing `ConvertibleTo`.